website/integrations/services/aws/index.md

```diff
@@ -71,78 +70,76 @@ return [
+-   On the right side, click **Actions** -> **Change identity source**
+-   Select **External Identity Provider**
+-   Under **Service Provider metadata** download the metadata file.
 
 Now go to your authentik instance, and perform the following steps.
 
--   Under _Providers_ create a new _SAML Provider from metadata_. Give it a name, and upload the metadata file AWS gave you.
--   Click _Next_. Give it a name, and close the file.
+-   Under **Providers**, create a new **SAML Provider from metadata**. Give it a name, and upload the metadata file AWS gave you.
+-   Click **Next**. Give it a name, and close the file.
 -   If you haven't done so yet, create an application for AWS and connect the provider to it.
--   Navigate to the provider you've just created, and then select _Edit_
--   Copy the _Issuer URL_ to the _Audience_ field.
--   Under _Advanced Protocol Settings_ set a _Signing Certificate_
+-   Navigate to the provider you've just created, and then select **Edit**
+-   Copy the **Issuer URL** to the **Audience** field.
+-   Under **Advanced Protocol Settings** set a **Signing Certificate**
 -   Save and Close.
--   Under _Related Objects_ download the _Metadata file_, and the _Signing Certificate_
+-   Under **Related Objects**, download the **Metadata file** and the **Signing Certificate**
 
 Now go back to your AWS instance
 
--   Under _Identity provider metadata_ upload both the the _Metadata_ file and _Signing Certificate_ that authentik gave you.
--   Click _Next_.
--   In your settings pane, under the tab _Identity Source_, click _Actions_ -> _Manage Authentication_.
--   Take note of the _AWS access portal sign-in URL_ (this is especially important if you changed it from the default).
+-   Under **Identity provider metadata**, upload both the **Metadata** file and **Signing Certificate** that authentik gave you.
+-   Click **Next**.
+-   In your settings pane, under the tab **Identity Source**, click **Actions** -> **Manage Authentication**.
+-   Note the AWS access portal sign-in URL (especially if you have customized it).
 
 Now go back to your authentik instance.
 
--   Navigate to the Application that you created for AWS and click _Edit_.
--   Under _UI Settings_ make sure the _Start URL_ matches the _AWS access portal sign-in URL_
+-   Navigate to the Application that you created for AWS and click **Edit**.
+-   Under **UI Settings** make sure the **Start URL** matches the **AWS access portal sign-in URL**.
 
 ## Caveats and Troubleshooting
 
--   Users need to already exist in AWS in order to use them through authentik. AWS will throw an error if it doesn't recognise the user.
--   In case you're stuck, you can see the SSO logs in Amazon CloudTrail -> Event History. Look for `ExtenalIdPDirectoryLogin`
-
-Note:
+-   Ensure users already exist in AWS for authentication through authentik. AWS will throw an error if the user is unrecognized.
```

Comment:
Ideally we ant to use the following syntax for Notes:

::: info
ADDYOURNOTEHERE
:::